### ISSUE TITLE
includePaths doesn't work?

### ISSUE THREAD

**Issue by Epskampie** [glromeo/esbuild-sass-plugin#44]
I'm trying to use `includePaths`, but can't get it to work. I've made a tiny reproduction repo: https://github.com/Epskampie/esbuild-sass-includepaths

As you can see `npm run sass:build` works, `npm run esbuild:build` error out with:
`error: Cannot find module 'color/_colors'`

Some relevant code:
```ts
// bundler.ts
import esbuild from 'esbuild';
import { sassPlugin } from 'esbuild-sass-plugin';

esbuild
    .build({
      plugins: [
        sassPlugin({
          includePaths: [ 'scss_utils', ],
        }),
      ],
      outdir: 'dist',
      entryPoints: ['src/a.scss'],
    })
    .catch((e) => console.error('esbuild', e.message));
```

```scss
// src/a.scss
@use 'colors';

body {
  color: colors.$red;
}
```

```scss
// scss_utils/_colors.scss
$red: red;
```

**Comment by Epskampie** [comment]
After reading the docs some more, it seems that `includePaths` is not actually supported, but I can [achieve something similar](https://github.com/Epskampie/esbuild-sass-includepaths/tree/fix_importmapper) with `importMapper`.

You may want to remove this line from the README.md: `The options passed to the plugin are a superset of the Sass Options.` as it lead to be believe all sass options are supported.

Feel free to close this, and thanks for making this plugin!

**Comment by glromeo** [comment]
@Epskampie I re-opened this issue because I actually refreshed my memory and I realized that **includePaths is supported** and passed through to the sass JS API. The real problem is that because at some point the importer which originally was optional became a default it [takes over](https://sass-lang.com/documentation/js-api/interfaces/LegacySharedOptions#importer) and fails instead of delegating to the next step. I will provide a fix for that and close this issue then.
Thank you for your sample project it helped me debug quite quickly!
